.expeditor/config.yml

```diff
@@ -44,6 +44,9 @@ merge_actions:
         - "Expeditor: Skip Omnibus"
         - "Expeditor: Skip All"
       only_if: bash:.expeditor/determine_version.sh
+  - bash:.expeditor/push-git-tag.sh:
```

Comment:
So the tag is on the remote when the omnibus build occurs, we'll want to run this prior to triggering the omnibus build.

Reply:
I'm presuming the `post_commit: true` here is redundant?